FEATURE/ElvaRivle/DfWCIPVU

ElvaRivle opened the pull request:
FEATURE/ElvaRivle/DfWCIPVU

--------------------------------------------------------------------
 MAIN
 * successfully connected to MongoDB
 * created User model
 * seeded default users to DB
 --------------------------------------------------------------------
 OTHER
 * added all necessary packages
 * loading connection string from .env file
 * listening and logging mongoose events 
 * NOT READY FOR MERGE
 --------------------------------------------------------------------

ElvaRivle committed:
FEATURE/ElvaRivle/DfWCIPVU

--------------------------------------------------------------------
 MAIN
 * successfully connected to MongoDB
 * created User model
 * seeded default users to DB
--------------------------------------------------------------------
 OTHER
 * added all necessary packages
 * loading connection string from .env file
 * listening and logging mongoose events
--------------------------------------------------------------------

becarselman commented:
Everything looks good. Good job!.  
I have one suggestion, to trigger seeders as script. So we will have command to do that for example "npm run seeders"
Also you can document that in readme file

melcidar reviewed (approved):
Everything looks good to me

ElvaRivle committed:
FEATURE/ElvaRivle/DfWCIPVU

--------------------------------------------------------------------
 MAIN
 * seeder converted to script
 * README added
--------------------------------------------------------------------
 OTHER
 * changed default users for seeding
--------------------------------------------------------------------
